cypress/pageobjects/rancher.po.ts

```diff
@@ -0,0 +1,497 @@
+            // cy.contains(target).parentsUntil('tbody', 'tr').find('td.col-badge-state-formatter').contains(valid ? 'Active' : 'Removing', { timeout: constants.timeout.provisionTimeout }).should('not.exist');
+            cy.contains(target).parentsUntil('tbody', 'tr').should('not.exist', { timeout: constants.timeout.downloadTimeout });
+        });
+
+    }
+
+    public createCloudCredential(cloud_credential: string, harvester_cluster_name: string){
+
+        this.visit_cloudCredential();
+      
+        cy.wait(1000).get(this.cloudCredential_page_createButton).click();
+ 
+        cy.get(this.cloudCredential_page_harvester).should('contain', 'Harvester').click();
+        
+        cy.get(this.cloudCredential_page_clusterName).type(cloud_credential);
+
+        cy.get('.vs__search').click().then(($list) => {
+            cy.contains(harvester_cluster_name).click();            
+        })
+        
+        cy.get(this.cloudCredential_page_confirmCreate).should('contain', 'Create').click();
+
+        cy.contains(cloud_credential);
+    }
+
+    public input_RKE2_Cluster_Content(rke2_cluster_attributes: any) {
+
+        this.visit_clusterManagement();
+        
+        cy.get(this.clusterManagement_page_create).click();
+
+        // Set RKE2 checkbox back to default
+        cy.get('span[class="label no-select hand active"]').then((el) => {
+            let active = el.text();
+            console.log('Current activate in: ', active)
+            cy.log(active);
+            if (active == 'RKE2/K3s'){
+                cy.get(this.clusterManagement_rke_selector).click();
+            }
+        })
+        
+        // toggle slide
+        cy.wait(1000).get(this.clusterManagement_rke_selector).click().then((el) => {
+
+            cy.get(this.clusterCreation_page_harvester).should('contain', 'Harvester').click();
+
+            // Input CPUs
+            cy.get(this.rke2Creation_page_cpus).clear().type(rke2_cluster_attributes.cpus);
+
+            // Input Memory
+            cy.get(this.rke2Creation_page_memory).clear().type(rke2_cluster_attributes.memory);
+
+            // Input Disk
+            cy.get(this.rke2Creation_page_disk).clear().type(rke2_cluster_attributes.disk);
+
+            // Select Namespace
+            cy.get(this.rke2Creation_page_namespaceCombo).click().then(($list) => {
+                cy.get(this.rke2Creation_page_namespaceOption).should('contain', rke2_cluster_attributes.namespace).click(); 
+            })
+
+            // Select Image
+            cy.get(this.rke2Creation_page_imageCombo).click().then(($list) => {
+                cy.get(this.rke2Creation_page_imageOption).should('contain', rke2_cluster_attributes.image).click();
+            })
+
+            // Select Network
+            cy.get(this.rke2Creation_page_networkNameCombo).click().then(($list) => {
+                cy.get(this.rke2Creation_page_networkNameOption).should('contain', rke2_cluster_attributes.network_name).click(); 
```

Comment:
```suggestion
            const networkSelect = new LabeledSelectPo('.labeled-select', `:contains("Network Name")`);
            networkSelect.select({option: rke2_cluster_attributes.network_name, selector: '.vs__dropdown-menu'});
```

Ditto, will be failed when there have other network options.

Reply:
Thanks for the instruction, change the menu drop down list selection using LabeledSelectPo in /utils/components/labeled-select.po

```
const networkSelect = new LabeledSelectPo('.labeled-select', `:contains("Network Name")`);
networkSelect.select({option: rke2_cluster_attributes.network_name, selector: '.vs__dropdown-menu'});
```

And also update the namespace and image drop down selection using the same manner